hw3/01-characters.js

```diff
@@ -0,0 +1,40 @@
+// url for the Thrones API
+const url = 'https://thronesapi.com/api/v2/Characters';
+const app = document.querySelector('section');
+
+// Add your code here
+const addToDOM = (character) => {
+  const card = document.createElement('div');
+  const img = document.createElement('img');
+  const body = document.createElement('div');
+  const fullName = document.createElement('h4');
```

Comment:
You're jumping to an h4 from an h1 which is giving you accessibility issues. I would make this h2.

Reply:
I always forget that, thanks for reminding me. I'll make the change.